.github/workflows/test-action-build-init.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         dest: ['']
         antsibull-docs-version:
           - ''  # there is no default so this will be ok
-          - 1.0.0
+          - 1.7.0
```

Comment:
I think the idea is to have 1.0.0 here as an older version.

How about keeping this here, and instead don't double the matrix size, but only add a few explicit matrix entries (`include:`) that cover he `intersphinx-links` parameter?